Duplicati/CommandLine/Commands.cs

```diff
@@ -267,7 +339,7 @@ public static int List(TextWriter outwriter, Action<Duplicati.Library.Main.Contr
                             options["version"] = v.ToString();
                         }
                     }
-                    else if (args[0].IndexOfAny(new char[] { '*', '?' }) < 0 && !args[0].StartsWith("[", StringComparison.Ordinal))
+                    else if (args[0].IndexOfAny(new char[] { '*', '?' }) < 0 && !args[0].StartsWith("[", StringComparison.Ordinal) && !args[0].StartsWith("@", StringComparison.Ordinal))

```

Comment:
Can we use `!ContainsWildcards` instead of `IndexOfAny` here?

Reply:
Fixed in latest commit.